app/controllers/legal_docs_controller.rb

```diff
@@ -0,0 +1,44 @@
+class LegalDocsController < ApplicationController
+  before_filter :find_legal_doc, except: [:index, :new, :create]
```

Comment:
That was not my understanding of the Index function that we talked about:
   Index = show ALL the legal docs we had ever made (history)
   Show = show the latest Terms and Privacy statement (from the footer links)

But, I will adjust the index and avoid the smart_listing altogether.